scripts/deployToDB.py

```diff
@@ -294,22 +340,38 @@ def print_summary(selector, final_report, dry_run=False):
         print("\n")
         print("#" * 50)
         mode_text = " (DRY RUN)" if DRY_RUN else ""
-        print("Running {} Definitions Updates{}".format(selector.capitalize(), mode_text))
-        final_report = update_diff_db(selector, ITEM_NAME, DRY_RUN)
+        print(
+            "Running {} Definitions Updates{}".format(selector.capitalize(), mode_text)
+        )
+        final_report = update_diff_db(selector, ITEM_NAME, DRY_RUN, VERBOSE)
 
         # Always show summary first (most important for users)
         print_summary(selector, final_report, DRY_RUN)
 
-        # Show detailed reports only when verbose flag is used
+        # Show detailed reports only when verbose flag is used (write to debug.log)
         if VERBOSE:
```

Comment:
verbose has to be enabled by default to check individual API status code - otherwise, we don't know if release was successful in deploying all definitions

Reply:
okay, intention of verbose is for local testing, if we want to use it in ci then we need to think differrently

Reply:
noise was coming as every definition is getting logged. we can filter the ones that are not applicable for updates (status=N/A) and can skip verbose feature